### PR TITLE
Adding target to docker-compose.yml file

### PR DESCRIPTION
**Pull request by jinder1s**
This would allow us to build service images with "docker-compose build". And changed images to point to openedx instead of kdmccormick dockerhub

### CODE REVIEW

File: docker-compose.yml [hunk 1]
```diff
@@ -35,11 +32,10 @@ services:
       ENABLE_DJANGO_TOOLBAR: 1
 
   worker:
-    # Uncomment this line to use the official catalog base image
-    # image: edxops/enterprise_catalog:devstack
+    image: openedx/enterprise_catalog:latest-devstack
     build:
       context: "."
-    image: kdmccormick96/enterprise-catalog  # TODO: Change to openedx.
+      target: devstack
```
Comment (kdmccormick) [inline docker-compose.yml]: the `image` key declares an image that can be pulled down from dockerhub and used.

the `build` key instructs docker-compose how to build the image locally.

I believe we'd only want one or the other--either it should be built locally (my preference) or it should get pulled down from dockerhub.

Comment (jinder1s) [reply docker-compose.yml]: I definitely also prefer build to pull, but am more on the side of giving both as an option. I will think on this for a bit

Comment (jinder1s) [reply docker-compose.yml]: Okay, I'm with you on just the build

Comment (jinder1s) [reply docker-compose.yml]: @kdmccormick this should be good to go now

**Review by kdmccormick — APPROVED** [review]

**Pull request merged**